Sources/VitaminCore/Components/Badge/VitaminBadgeVariant.swift

```diff
@@ -53,3 +53,20 @@ public enum VitaminBadgeVariant: CaseIterable {
         return 0
     }
 }
+
+public extension VitaminBadgeVariant {
+    var name: String {
+        switch self {
+        case .standard:
+            return "Standard"
+        case .brand:
+            return "Brand"
+        case .reversed:
+            return "Reversed"
+        case .accent:
+            return "Accent"
+        case .alert:
+            return "Alert"
+        }
+    }
+}
```

Comment:
It seems to be used only in the UIKit showcase, so please move it to the `Showcase/Application/UIKit/Components/Badge` folder.

Reply:
done